sqlglot/generator.py

```diff
@@ -38,8 +38,18 @@ class Generator:
     """
 
     TRANSFORMS = {
+        exp.AnonymousProperty: lambda self, e: self.property_sql(e),
+        exp.AutoIncrementProperty: lambda self, e: f"AUTO_INCREMENT={e.text('value')}",
```

Comment:
should we do self.sql(e, 'value') instead of all the text references, and then the quotes vs no quote is automatic

Reply:
Good call that helps a lot